packages/wrangler/src/config/validation.ts

```diff
@@ -155,6 +159,18 @@ export function normalizeAndValidateConfig(
 				rawConfig
 			);
 			diagnostics.addChild(envDiagnostics);
+		} else if (isPagesConfig(rawConfig)) {
```

Comment:
Do we need to run the `normalizeAndValidateEnvironment()` in this case at all?
We already have all the configuration from the top level validation above.

How about instead, on line [174](https://github.com/cloudflare/workers-sdk/pull/5234/files#diff-0ec779a40744e36b45a9b28dee06739afa8aa2f00bb9388b68556e7fd87ab99eR174) we just change it to 

```
} else if (!isPagesConfig(rawConfig)) {
```